# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: intl/l10n/Localization.h
# Commit: 595529cd906f
# Full Hash: 595529cd906f94cdb2bc12a7a40072db0c07198e
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-03 22:01:19
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part9: Update DOMLocalization to match Localization constructor. r=nika,emilio
#   
#   Depends on D113691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113692
# ==============================================================================

diff -r 9f3e2963d925 -r 595529cd906f intl/l10n/Localization.h
--- a/intl/l10n/Localization.h	Tue Aug 03 05:52:03 2021 +0000
+++ b/intl/l10n/Localization.h	Tue Aug 03 05:52:03 2021 +0000
@@ -71,10 +71,10 @@
       const dom::Sequence<dom::OwningUTF8StringOrL10nIdArgs>& aKeys,
       nsTArray<dom::Nullable<dom::L10nMessage>>& aRetVal, ErrorResult& aRv);
 
-  void AddResourceId(const nsAString& aResourceId);
-  uint32_t RemoveResourceId(const nsAString& aResourceId);
-  void AddResourceIds(const nsTArray<nsString>& aResourceIds);
-  uint32_t RemoveResourceIds(const nsTArray<nsString>& aResourceIds);
+  void AddResourceId(const nsACString& aResourceId);
+  uint32_t RemoveResourceId(const nsACString& aResourceId);
+  void AddResourceIds(const nsTArray<nsCString>& aResourceIds);
+  uint32_t RemoveResourceIds(const nsTArray<nsCString>& aResourceIds);
 
   void SetAsync();
   bool IsSync();